### PR TITLE
Fixing issues in the TCP Monitoring page

### PR DESCRIPTION
**Pull request by samgnaniah**
Fixes https://github.com/wso2/docs-apim/issues/4106

### CODE REVIEW

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md [hunk 1]
```diff
@@ -4,27 +4,26 @@ Users can view and monitor the messages passed along a TCP-based conversation us
 
 TCPMon is not dependent on any third party libraries. Its user interface is based on a swing UI and works on almost all platforms that support Java.
 
--   [Starting TCPMon](_Starting_TCPMon_)
--   [Message Monitoring with TCPMon](_Message_Monitoring_with_TCPMon_)
--   [Other Usages of TCPMon](_Other_Usages_of_TCPMon_)
+-   Starting TCPMon
+-   Message Monitoring with TCPMon
+-   Other Usages of TCPMon
```
Comment (Mariangela) [inline en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md:9]:
```suggestion
```

Comment (Mariangela) [reply en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md:9]: We dont need this as the TOC is there on the right side.

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md [hunk 2]
```diff
@@ -4,27 +4,26 @@ Users can view and monitor the messages passed along a TCP-based conversation us
...
-
-# Starting TCPMon
+## Starting TCPMon
 
 TCPMon is available in the `<PRODUCT_HOME>/bin` directory of any WSO2 Carbon based product distribution. Alternatively, you can download TCPMon from Apache and run the tool.
 
--   [Running TCPMon (from Carbon product pack)](#StartingTCPMon-RunningTCPMon(fromCarbonproductpack))
--   [Running TCPMon (downloaded from Apache)](#StartingTCPMon-RunningTCPMon(downloadedfromApache))
+-   Running TCPMon (from Carbon product pack
+-   Running TCPMon (downloaded from Apache
```
Comment (Mariangela) [inline en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md:16]:
```suggestion
```

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md [hunk 3]
```diff
@@ -64,33 +63,31 @@ To download TCPMon from Apache and run the tool:
...
+-   Sending Requests for Web Services
+-   As a Proxy
+-   Advanced Settings
 
 #### Sending Requests for Web Services
 
 TCPMon can also be used as a request sender for Web services. The request SOAP message can be pasted on the send screen and sent directly to the server.
 
-![]({{base_path}}/assets/attachments/45946410/46206514.png)
+<a href="{{base_path}}/assets/img/administer/tcpmon_screen4.png"><img src="{{base_path}}/assets/img/administer/tcpmon_screen4.png" alt="TCPMon sending requests" name="TCPMon sending requests" width="520"></a>
```
Comment (Mariangela) [inline en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md:78]: This screen including the others are outdated.

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md [hunk 4]
```diff
@@ -64,33 +63,31 @@ To download TCPMon from Apache and run the tool:
...
-# Other Usages of TCPMon
-
-TCPMon is primarily used for message monitoring. Additionally, TCPMon can also be used for sending requests to web services and as a proxy service. Refer [Starting TCPMon](https://docs.wso2.com/display/Carbon440/Starting+TCPMon) for details on how to start the tool.
-
--   [Sending Requests for Web Services](#OtherUsagesofTCPMon-SendingRequestsforWebServices)
--   [As a Proxy](#OtherUsagesofTCPMon-AsaProxy)
--   [Advanced Settings](#OtherUsagesofTCPMon-AdvancedSettings)
+-   Sending Requests for Web Services
+-   As a Proxy
+-   Advanced Settings
```
Comment (Mariangela) [inline en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md:72]:
```suggestion
```

**Review by Mariangela — APPROVED** [review]
@samgnaniah , please check on this page as Tharindu said that this page together with the other TCPMon pages are not applicable.

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md:9]: Yes. +1. Let's remove.

File: en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md (hunk 3 above)
Comment (samgnaniah) [reply en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md:70]: Yes. For now let's merge and see what needs to be done with this page.

**Pull request merged**